yktracec/src/jitmodbuilder.cc

```diff
@@ -1314,8 +1314,8 @@ class JITModBuilder {
                 I++;
                 CurInstrIdx++;
               }
-              assert(InpTrace.Length() > 2);
-              if (Idx == InpTrace.Length() - 2) {
+              assert(InpTrace.Length() > 1);
```

Comment:
I believe this assert is no longer needed. It probably was incorrect in the first place as we only want to make sure that `InpTrace.Length() - 1 ` is valid. But we already know that since we wouldn't have entered the loop if the length was 0.

Reply:
Agreed. Removed in b7cb83ae3252d5401e3d5fe706b7b1fadfddbff7.